js/block-editor.js

```diff
@@ -0,0 +1,26 @@
+var editPost = wp.data.select( 'core/edit-post' ), lastIsSaving = false;
+
+wp.data.subscribe(
+	function() {
+			var isSaving = editPost.isSavingMetaBoxes();
```

Comment:
Indentation mistake?

Reply:
Thanks!  I think this got fixed in 5cc768a78552c7d53ab4338a60d8bcad664802a1 ... does it look correct to you now?